packages/data/addon/serializers/elide-facts.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * A serializer for an Elide response
+ */
+
+import EmberObject from '@ember/object';
+import NaviFactSerializer, { ResponseV1 } from './fact-interface';
+import { AsyncQueryResponse, RequestV1 } from 'navi-data/adapters/fact-interface';
+
+export default class ElideFactsSerializer extends EmberObject implements NaviFactSerializer {
+  /**
+   * Normalizes elide response into ResponseV1
+   * @param payload {AsyncQueryResponse} - payload to normalize
+   * @param request {RequestV1} - request for response payload
+   * @returns {ResponseV1}
```

Comment:
```suggestion
   * @param payload {AsyncQueryResponse} - payload to normalize
   * @param request {RequestV1} - request for response payload
   * @returns {ResponseV1|undefined}
```

Reply:
Are types necessary in the JSDoc when it's already a .ts file?

Reply:
If we used TSDoc we can skip types https://github.com/microsoft/tsdoc